definitions/npm/redux-reactors_v1.x.x/flow_v0.95.x-/redux-reactors_v1.x.x.js

```diff
@@ -0,0 +1,55 @@
+/* Helper types from 'redux'
```

Comment:
I'd put a biiiiig comment here saying they're copied, and indicate it as a complete section, so show where the copying stops.